recipes/plumpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "plumpy" %}
+{% set version = "0.7.9"%}
```

Comment:
Yes, the 0.7.9 version is the one we need for our materials science package at the moment.
Like for `aldjemy` I will be happy to add another recipe for the latest version after that.